### DISCUSSION TITLE
Plot list of PolyData or Line as a mesh with a same name

### DISCUSSION THREAD

**Discussion by AliRezaBeigy** [pyvista/pyvista#4520] [Q&A] [answered]
Hi
I'm looking for a way to plot the edge between point clouds as mesh with a same mesh name(to update the edges with that name, some of the edges may be removed after a while)

How can I do something like the one below?
```python
lines = list(map(lambda edge: pv.Line(graph.nodes[edge[0]]['pos'], graph.nodes[edge[1]]['pos']), graph.edges))
plotter.add_mesh(lines, color=[37, 109, 161], line_width=10, name='edges')
```

Currently, the above code has an error, and add_mesh does not accept a list of Line

**Comment by MatthewFlamm** [comment, marked as answer]
Does what you want to do work if you make it a `MultiBlock` and then use `plotter.add_composite`?

```py
lines = pv.MultiBlock(list(map(lambda edge: pv.Line(graph.nodes[edge[0]]['pos'], graph.nodes[edge[1]]['pos']), graph.edges)))
plotter.add_composite(lines, ...)
```
